index.js

```diff
@@ -149,10 +164,30 @@ class I18n {
      * @return {self}
      *
      * @example
-     * i18n.onmiss((key) => logMissingKeyEvent({key: key.replace(/\W/g, '_'), })
+     * i18n.onmiss((key) => logMissingKeyEvent({key})
      */
     onmiss(callback) {
-        typeof callback === 'function' && this[MISSING].push(callback);
+        if (typeof callback === 'function') {
```

Comment:
You can afford to validate once (either here or on execution).

Reply:
Yes, implemented better